src/utilities/borders.scss

```diff
@@ -23,17 +23,26 @@ $edges: (
   left: (bottom-left, top-left)
 );
 
+/* Responsive gray borders */
 @each $breakpoint, $variant in $responsive-variants {
   @include breakpoint($breakpoint) {
-    /* Add a gray border */
+    /* Add a gray border on all sides at/above this breakpoint */
+    .border#{$variant}-1      { border: $border !important; }
```

Comment:
Is there a specific reason for adding `-1` at the end? Is it a hint that the border has a `1px` width?

Otherwise it feels more intuitive to just have `border-(sm|md|lg|xl)` and `border-(sm|md|lg|xl)-(top|right|bottom|left)`.